django/core/checks/templates.py

```diff
@@ -33,3 +41,32 @@ def check_string_if_invalid_is_string(app_configs, **kwargs):
             error.msg = error.msg.format(string_if_invalid, type(string_if_invalid).__name__)
             errors.append(error)
     return errors
+
+
+@register(Tags.templates)
+def check_for_mulitple_libraries_with_the_same_name(app_configs, **kwargs):
+    errors = []
+
+    libraries = defaultdict(list)
+    candidates = ['django.templatetags']
+    candidates.extend(f'{app.name}.templatetags' for app in apps.get_app_configs())
+
+    for candidate in candidates:
```

Comment:
This code is based off `django.template.backends.django.get_installed_libraries`

https://github.com/django/django/blob/ae89daf46f83a7b39d599d289624c3377bfa4ab1/django/template/backends/django.py#L100